docs/reference-architectures/hybrid-networking/vnet-topology-hub-spoke.md

```diff
@@ -0,0 +1,430 @@
+
+* **VPN connection**. The connection has properties that specify the connection type (IPSec) and the key shared with the on-premises VPN appliance to encrypt traffic.
+
+* **Hub virtual network (VNet)**. Azure VNet used as a hub in a hub-spoke topology. The hub is used as a central point of connectivity to your on-premises network, and a place to host services that can be consumed by different workloads hosted in spoke VNets.
+
+    * **Gateway subnet**. The virtual network gateways are held in the same subnet.
+
+    * **Shared services subnet**. A subnet in the hub VNet used to host services that can be shared among all spokes, such as DNS, AD DS, among others.
+
+* **VNet peering**. You can connect two VNets in the same Azure region using a peering connection. Once peered, paired VNets exchange traffic by using the Azure backbone, without the need of a router. [Virtual networking peering][vnet-peering] connections are non-transitive, low latency connections between VNets.
+
+* **Spoke VNet**. Azure VNet used as a spoke in a hub-spoke topology. Spokes can be used to isolate wrokloads in their VNets, that can be managed separatly from other spokes. Each workload might include multiple tiers, with multiple subnets connected through Azure load balancers. For more information about the application infrastructure, see [Running Windows VM workloads][windows-vm-ra] and [Running Linux VM workloads][linux-vm-ra].
+
+## Recommendations
+
+The following recommendations apply for most scenarios. Follow these recommendations unless you have a specific requirement that overrides them.
+
+### VNet and GatewaySubnet
+
+Create the ExpressRoute virtual network gateway and the VPN virtual network gateway in the same VNet. This means that they should share the same subnet named *GatewaySubnet*.
+
+If the VNet already includes a subnet named *GatewaySubnet*, ensure that it has a /27 or larger address space, so that you can add both an ExpressRoute gateway, and a VPN gateway, for [higher availability][hybrid-ha].
+
+### VNet peering
+
+VNet peering is a non-transitive relationship between two VNets. If you require spokes to connect to each other, consider adding a separate peering connection between those spokes.
+
+However, if you have several spokes that need to connect with each other, you will run out of possible peering connections very quickly due to the [limitation on number of VNets peerings per VNet][vnet-peering-limit]. In this scenario, consider using user defined routes (UDRs) to force traffic destined to a spoke to be sent to the gateway at the hub VNet. This will allow the spokes to connect to each other, as shown below.
+
+![[1]][1]
+
+> [!NOTE]
+> You can use the sample deployment provided in this document to add these UDRs and make the peering connections transitive.
+
+To allow traffic to flow through the hub from one spoke to another, you need to:
+
+  - Configure the VNet peering connection in the hub to **allow gateway transit**.
```

Comment:
I'll take that back. There is nothing in docs.microsfot.com that explain that. the only link I found was https://docs.microsoft.com/en-us/cli/azure/network/vnet/peering 
I'll leave it as is for now, and raise the issue with the PG.